platform-sdk/swirlds-unit-tests/core/swirlds-platform-test/src/main/java/com/swirlds/platform/test/consensus/ShadowGraphEventObserver.java

```diff
@@ -51,7 +53,7 @@ public ShadowGraphEventObserver(@NonNull final ShadowGraph shadowGraph) {
      */
     @Override
     public void consensusRound(final ConsensusRound consensusRound) {
-        shadowGraph.expireBelow(consensusRound.getGenerations().getMinRoundGeneration());
+        shadowGraph.updateNonExpiredEventWindow(consensusRound.getNonAncientEventWindow());
```

Comment:
This class is no longer used outside of tests

Reply:
Yup, I'm aware of that. That's why I moved this into the test directory. It's currently used by `TestIntake`, which is why I did not remove it entirely.